README.md

```diff
@@ -1,63 +1,100 @@
 # Cartography
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](LICENSE)![Version](https://img.shields.io/nuget/v/Cartography.DynamicMap?style=flat-square)![Downloads](https://img.shields.io/nuget/dt/Cartography.DynamicMap?style=flat-square)   
 
-Nventive solution for mobile app map.
-
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+Cartography provides you 3 map related libraries for iOS, Android and Windows using native map of each platform.
 
 ## Getting Started
 
 ### **Samples**
 - Clone project a sample is available.
-- For seeing samples, build and install app with VS on the desire device (Android, IOS, or UWP)
+- For seeing samples, build and install app with VS on the desire device (Android, IOS, or UWP).
+
+### API Key (not needed for mapService)
+- For Google Maps (Android), you need to [create your Api Key](https://developers.google.com/maps/documentation/android-sdk/get-api-key).
+- For Bing Maps, you need a [Bing Api Key](https://learn.microsoft.com/en-us/bingmaps/getting-started/bing-maps-dev-center-help/getting-a-bing-maps-key) to remove warning.
+
+Set it in : sample.shared => Constants.
+
+```csharp
+public class GoogleMaps
+{
+	//TODO: Get new API Key
+	public const string ApiKey = "";
+}
+```
+
+### **DynamicMap**
+
+### Permission
+- Maps doesn't required to ask permission to the user.
```

Comment:
Permissions for what? Why are we mentioning this in the Getting Started section if it's not even needed?

It's also a bit confusing because the next nested section is about setting up permissions on Android, which seem to be required.